client/src/nonblocking/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod rpc_client;
```

Comment:
I went with `nonblocking` as the mod name instead of:
* `non_blocking` -- usage of nonblocking and non-blocking seem to about equal
* `async` -- nice, but that's a keyword so there's a lot of `r#sync`-escaping required and that gets old

Also naming the async RpcClient as `AsyncRpcClient` or similar causes too much client code churn.

Reply:
The only other idea I've had so far is to riff on `await`, like `awaiting` eg. Fewer characters, but feels weirdish